test/scripts/e2e-kind.sh

```diff
@@ -17,6 +17,7 @@ Disruptive
 # FEATURES NOT AVAILABLE IN OUR CI ENVIRONMENT
 \[Feature:Federation\]
 should have ipv4 and ipv6 internal node ip
+should have ipv4 and ipv6 node podCIDRs
```

Comment:
what is this test? why is it unrelated?

Reply:
because the podCIDRs are created by the nodeipam controller in the controller-manager and ovn doesn't use it.
Also, this field is something that upstream wants to deprecate ...

Reply:
I didn't check why they are failing in this PR, is probably something for my TODO in upstream